src/components/WorkChart.astro

```diff
@@ -0,0 +1,17 @@
+---
+type Props = {
+  img: string;
```

Comment:
This is ok but we want components to be a bit more specific when it comes to props, instead of img, change it to imgSrc, and also add imgAlt to be able to send the Alt text to the image tag as well.